src/resource_clients/actor.ts

```diff
@@ -459,3 +462,50 @@ export interface ActorDefinition {
     maxMemoryMbytes?: number;
     usesStandbyMode?: boolean;
 }
+
+type CommonActorPricingInfo = {
+    /** In [0, 1], fraction of pricePerUnitUsd that goes to Apify */
+    apifyMarginPercentage: number;
+    /** When this pricing info record has been created */
+    createdAt: Date;
+    /** Since when is this pricing info record effective for a given Actor */
+    startedAt: Date;
+    notifiedAboutFutureChangeAt?: Date;
+    notifiedAboutChangeAt?: Date;
+    reasonForChange?: string;
+}
+
+export type FreeActorPricingInfo = CommonActorPricingInfo & {
+    pricingModel: 'FREE';
```

Comment:
Maybe a string enum for the `pricingModel` variants? I've got no hard opinion here though